src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -17,6 +17,36 @@
           }
         ]
       },
+      "logicalLocations": {
+        "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()": [
+          {
+            "name": "app",
+            "kind": "android-module"
```

Comment:
OK, having general `type` as well as specific `class` is confusing. Omit `type`, but keep `function`.